core/src/main/resources/lib/hudson/project/configurable.jelly

```diff
@@ -24,20 +24,10 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <!-- Displayed by projects (assignable to ParameterizedJobMixIn.ParameterizedJob) which can be configured and built. -->
-<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:st="jelly:stapler">
     <j:if test="${it.buildable}">
-        <j:set var="id" value="${h.generateId()}"/>
-        <l:task href="${url}/build?delay=0sec" icon="icon-clock icon-md" onclick="${it.parameterized?null:'return build_' + id + '(this)'}" permission="${it.BUILD}" post="${!it.parameterized}" title="${it.buildNowText}"/>
-        <script>
-            function build_${id}(a) {
-                fetch(a.href, {
-                  method: 'post',
-                  headers: crumb.wrap({}),
-                })
-                hoverNotification('${%Build scheduled}',a.parentNode);
-                return false;
-            }
-        </script>
+        <st:adjunct includes="lib.hudson.project.configurable.configurable"/>
+        <l:task href="${url}/build?delay=0sec" icon="icon-clock icon-md" permission="${it.BUILD}" post="${!it.parameterized}" data-callback="lib_hudson_project_configurable_build_now_callback" data-build-success="${%Build scheduled}" data-parameterized="${it.parameterized}" title="${it.buildNowText}"/>
```

Comment:
The only purpose of the callback seems to be to change the message from `Done` to `Build scheduled`. 
So if we generalize the success message for the post we don't need the callback and the js here.
But I like the idea with the callback

Reply:
>  So if we generalize the success message for the post we don't need the callback and the js here.

I didn't want to remove functionality unless it really was necessary or required crazy workarounds. The solution here seems generally sensible and supportable.

Doing it this way also provides an additional nontrivial example that can help plugin developers adapt their own code.